pages/manuals/[[...pageUrl]].jsx

```diff
@@ -97,50 +98,16 @@ function GetPage({ tree, page, catalogPage }) {
     }, [children, catalogId])
 
     useEffect(() => {
-        if (!children || !pageUrl) {
```

Comment:
I move it to [utils/getManualToc.js](https://github.com/ekaterinburgdev/guides/pull/41/files#diff-9bfd8564dd715ccf367724eacffe4fe28b87b0271df1502989c1fc767ff7c868) and not edit this file